tests/integ/modin/frame/test_apply.py

```diff
@@ -603,6 +603,7 @@ def g(v):
 ]
 
 
+@pytest.mark.modin_sp_short_regress
```

Comment:
this test seems simply raises an NotImplementedError, should we at least get one test point that runs?

Reply:
right this one does not need to be in.